guides/common/modules/proc_creating-a-local-source-for-a-custom-file-type-repository.adoc

```diff
@@ -29,7 +29,7 @@ endif::[]
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} python3.11-pulp_manifest
+# {project-package-install} pulp-manifest
```

Comment:
Unfortunately, so far Satellite hasn't changed the packaging.

@ekohl Is there a ticket for it? Who can we ping about this?

Reply:
https://issues.redhat.com/browse/SAT-24998

Reply:
@odilhao?

Reply:
@Lennonka I'm the one to ping :smile: